src/mainwindow.cpp

```diff
@@ -112,6 +106,21 @@ void MainWindow::ClearRecentFileMenu()
     Ui_MainWindow::menuRecent_files->addSeparator();
 }
 
+void MainWindow::UpdateTheme()
+{
+    QString themeName = m_options.getTheme();
+
+    ThemeUtils::SwitchTheme(themeName, this);
+
+    // Update toolbar Icons theme
+    this->actionTail_current_tab->setIcon(QIcon(ThemeUtils::GetThemedIcon(":/ctx-livecapture.png")));
+    this->actionHighlight_only_mode->setIcon(QIcon(ThemeUtils::GetThemedIcon(":/ctx-highlight.png")));
+    this->actionClear_all_events->setIcon(QIcon(ThemeUtils::GetThemedIcon(":/ctx-clear.png")));
+    this->actionShow_summary->setIcon(QIcon(ThemeUtils::GetThemedIcon(":/ctx-summary.png")));
+    this->actionRefresh->setIcon(QIcon(ThemeUtils::GetThemedIcon(":/ctx-refresh.png")));
+    this->actionFind->setIcon(QIcon(ThemeUtils::GetThemedIcon(":/ctx-find.png")));
```

Comment:
There is a minor bug where the Find button is enabled when there is no file open. 
actionFind needs to be disabled in UpdateMenuAndStatusBar() below. 
It's not noticed before because the entire Find menu is disabled so the actionFind was not visibly accessible.